scripts/generateANTLRClasses.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# Call this script from the repository's root directory './scripts/generateANTLRClasses.sh'
+
+# Define the paths and package information
+GRAMMAR_DIR="src/main/resources/grammars"
+GRAMMAR_FILE="CExpression.g4"
+PACKAGE="org.variantsync.diffdetective.feature.antlr"
+OUTPUT_DIR="$(pwd)/src/main/java/org/variantsync/diffdetective/feature/antlr"
+
+cd $GRAMMAR_DIR
+
+# Call ANTLR to generate the lexer and parser
+antlr4 -o $OUTPUT_DIR -package "$PACKAGE" "$GRAMMAR_FILE"
```

Comment:
Ah, I though that DiffDetective should not be used under Windows because the analyzed repos are not all supported by Window's file system (e.g. the linux kernel cannot be cloned).

Reply:
solved by #107